### DISCUSSION TITLE
2023-12-18 MESH CoP Meeting Notes

### DISCUSSION THREAD

**Discussion by hrynkiwc** [MESH-Model/MESH-Dev#32] [CoP Meetings]
**1. MESH Developers Sub-group; Lead: Al P.**
- MESH code revisions are taking place to incorporate GWF updates, with a release expected towards the end of January.
- GWF is using two versions of MESH exist: one for gridded setups and a different version for vector setups. A third version may exist with updates for Mountain MESH. Yukon is using a gridded setup. Vector-based setups are not available for the Saskatchewan and Nelson rivers, or for the Mackenzie.It would be helpful to converge the two versions so that everyone is using the same version. A student may be hired through PPWB to conduct evaluations.
- A shift to a software services platform would be favorable for running MESH instead of relying on personal drives.
- Look into possible overlap of FloodNet/floodplain mapping projects.
- All of the GWF setups use CLASS but it would be interesting to explore using SVS.
- It’s important to tie together different research products such as HYPE, CIROH, SUMMA and others.
- ECCC is committed to supporting maintenance of the MESH framework, while innovation is the focus of academia, with the goal of building robust forecasting tools.

**2. Draft Terms of Reference Document; Lead: Charmaine H. and Dan P.**
MESH CoP Terms of Reference discussion notes: 

_Purpose:_
- Include reference to a climate change context and long term flow projections, especially for irrigation, which is particularly relevant for Alberta.
- Set up a standard for Canada.
- Include information on process representation.

_Scope:_
   - Consider the overlap with other entities and scientific developments.
    - It is crucial to keep everyone involved and updated on ongoing or new developments and technical components.

_Authority:_
- Establishing a formal steering committee could help with making decisions on priorities and assessments, ensuring representation from ECCC and developers.

_Meetings:_
- Meetings to occur at six-week intervals, with more frequent meetings for developers.

_Communication methods and platforms:_
- Suggestions include email, online meetings, GitHub, and GCcollab.

    
**5. Action Items and Person Responsible**
- Develop a Terms of Reference document for the MESH CoP; Bruce D., Dan P, Charmaine H.
- Develop a strategy for MESH development and updating.; Bruce D, Dan P., Al P.
- Begin scheduling MESH developer meetings to occur every other week, at least until the end of January.; Al P. and Dan P.

**6. Next meeting; Lead: Dan P.**
- The next MESH CoP meeting is set for Monday, January 29 at 14:00 CST (13:00 MST, 13:00 YST)

**Comment by kasra-keshavarz** [comment]
Thank you so much @hrynkiwc!
